ccf_transaction_processor/scripts/configure_ccf_network.py

```diff
@@ -99,15 +145,16 @@ def add_user_script(client, options, config) :
 
 # -----------------------------------------------------------------
 def Main() :
-    default_config = os.path.join(CCF_Etc, 'cchost.toml')
     parser = argparse.ArgumentParser(description='Script to enable the CCF network')
 
     parser.add_argument('--logfile', help='Name of the log file, __screen__ for standard output', type=str)
     parser.add_argument('--loglevel', help='Logging level', default='WARNING', type=str)
-
-    parser.add_argument('--ccf-config', help='Name of the CCF configuration file', default = default_config, type=str)
-    parser.add_argument('--member-name', help="Name of the member adding the user", default = "member0", type=str)
-    parser.add_argument('--user-name', help="Name of the user being added", default = "user0", type=str)
+    parser.add_argument('--ccf-config-path', help='Path to locate the ccf config file', default = CCF_Etc, type=str)
+    parser.add_argument('--ccf-config', help='Name of the CCF configuration file', required=True, type=str)
+    parser.add_argument('--member-name', help="Name of the member adding the user", default = "memberccf", type=str)
```

Comment:
any reason why this requires two parameters? we can just put in a full path if it isn't in the expected location

Reply:
Thank you, only reason was to make it easier for the user. The path has a default, while the config file has no default (if we will use multiple tomls as explained above). So this way the user only needs to provide the name of the config as the required option rather than the whole path as well.

Reply:
this is another small nit... but we are making more complex a service that we should almost never change from the default installation (a service we don't need to test in unique configurations except from a client's view). i would prefer to see service deployment be as simple as it can possibly be (that is cut down the files, cut down the number of places to configure, cut down the options). the generality of CCF doesn't really help us to advance PDO.